### PR TITLE
Remove redundent code and fetch brick utilization from etcd in collectd plugin(WIP)

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
bugzilla: 1637977
tendrl-bug-id:Tendrl/node-agent#852

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @anmolbabu @nthomas-redhat removed please review

File: tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py [hunk 1]
```diff
@@ -49,262 +51,57 @@ def __init__(self):
...
-                stdin=open(os.devnull, "r"),
-                stdout=subprocess.PIPE,
-                stderr=subprocess.PIPE,
-                close_fds=True
-            )
-            stdout, stderr = cmd.communicate()
-            if stderr:
-                collectd.error(
-                    'Failed to fetch lvs. Error %s' % stderr
+            bricks = self.etcd_client.read(
```
Comment (anmolbabu) [inline tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py]: IMHO I wouldn't want to rely on etcd for anything...
The plugins in my view should have independent(As much as possible) existence immaterial of presence or absence tendrl..(If I remember correctly, there were plans of moving the plugins into a new repo under gluster org )...
For that matter, I am even ok with importing a tendrl module here(if replicating logic is duplication of efforts and considerably tedious) but relying on existence of functional tendrl stack(is somehow not convincing to me)
But its possible that things have changed now... So, my review would probably be stale now ;)

Comment (GowthamShanmugam) [reply tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py]: yes, i agree with your comment, also this PR also will increase the number of etcd calls also,  This PR is just optional. for now we don't want to take this PR .

Comment (GowthamShanmugam) [reply tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py]: Thanks for your review @anmolbabu :)

Comment (anmolbabu) [reply tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py]: Np :)

**Comment by shtripat** [comment]
@GowthamShanmugam I would suggest you abandon this PR.

**Comment by GowthamShanmugam** [comment]
For now this PR is not required, so i am closing this

**Pull request closed without merging**